activejob/lib/active_job/arguments.rb

```diff
@@ -24,16 +24,16 @@ class SerializationError < ArgumentError; end
   module Arguments
     extend self
     # :nodoc:
-    TYPE_WHITELIST = [ NilClass, String, Integer, Float, BigDecimal, TrueClass, FalseClass ]
+    PERMITTED_TYPES = [ NilClass, String, Integer, Float, BigDecimal, TrueClass, FalseClass ]
 
-    # Serializes a set of arguments. Whitelisted types are returned
+    # Serializes a set of arguments. Permitted types are returned
```

Comment:
The constant name is fine, but I think this is too vague for API documentation: absent any clarification, "whitelisted" correctly implied a short cautiously constructed set of allowed values, whereas "permitted" implies "not [blacklisted]". "Intrinsic types that can safely round-trip" might be a better phrasing of what the documentation currently says... though I incidentally notice this documentation makes no mention of the new custom serialization support, which may rearrange its emphasis and wording more substantially.